### PR TITLE
Add javadocs to readme

### PR DESCRIPTION
**Pull request by MiniDigger**

### CODE REVIEW

**Comment by FormallyMyles** [comment]
👍 useful

**Comment by AlfieC** [comment]
IMO `paper` should be `Paper` and `api` should be `API`.

**Comment by MiniDigger** [comment]
(should be squashed while merging)

**Comment by AlfieC** [comment]
You could squash it yourself - `git pull` the patch branch, squash them and `push --force`.

**Comment by MiniDigger** [comment]
@AlfieC that would force me to check out the repo onto my pc. I did this using the online visual editor. that comment was just to tell who ever will be merging this to use the "squash and merge" and not the "create a merge commit" merge option.

**Comment by zachbr** [comment]
May as well fix paper's capitalization directly above, then properly capitalize the additions.

As @AlfieC suggests,

> IMO `paper` should be `Paper` and `api` should be `API`.

File: README.md [hunk 1]
```diff
@@ -8,7 +8,8 @@ High performance Spigot fork that aims to fix gameplay and mechanics inconsisten
 
 Documentation
 ------
-Access the paper docs here: [paper.readthedocs.io](https://paper.readthedocs.io/)
+Access the paper docs here: [paper.readthedocs.io](https://paper.readthedocs.io/)
```
Comment (kashike) [inline README.md]: `paper` -> `Paper`

File: README.md [hunk 2]
```diff
@@ -8,7 +8,8 @@ High performance Spigot fork that aims to fix gameplay and mechanics inconsisten
 
 Documentation
 ------
-Access the paper docs here: [paper.readthedocs.io](https://paper.readthedocs.io/)
+Access the paper docs here: [paper.readthedocs.io](https://paper.readthedocs.io/)  
+Access the paper api javadocs here: [paperdocs.emc.gs](https://paperdocs.emc.gs)
```
Comment (kashike) [inline README.md]: `paper` -> `Paper`
`api` -> `API`

**Review by kashike — CHANGES_REQUESTED** [review]

**Comment by AlfieC** [comment]
I'd recommend adding the [CI-SKIP] tag too - no need to waste a CI build on a readme change.

**Comment by MiniDigger** [comment]
Last commit addressed the capitalisation @kashike

**Review by kashike — APPROVED** [review]

**Pull request merged**